src/User.php

```diff
@@ -4815,7 +4815,8 @@ public function showItems($tech)
 
                 foreach ($item_iterator as $data) {
                     $cansee = $item->can($data["id"], READ);
-                    $link   = $data["name"];
+                    // Not all item types have a name
+                    $link   = $data["name"] ?? null;
```

Comment:
Instead of displaying an empty value, we may use the declared name field of the item.
```suggestion
                    $link   = $data[$item->getNameField()];
```

Reply:
If the value is empty or if IDs are visible, it will format the link text as '%1$s (%2$s)'.
It seems like another way to fix it would be to use the `getNameID` function and remove the duplicated logic.